Authentication/readme.md

```diff
@@ -0,0 +1,110 @@
+# Juntos Auth Service
+
+## This Authentication module will provide basic auth functionality as follows:-
+
+- signIn
```

Comment:
Thanks for adding the code snippets. Can you document these APIs (list them out and add a one liner explaining these.